Include/object.h

```diff
@@ -861,6 +861,15 @@ static inline int PyType_CheckExact(PyObject *op) {
 #  define PyType_CheckExact(op) PyType_CheckExact(_PyObject_CAST(op))
 #endif
 
+#if !defined(Py_LIMITED_API)
```

Comment:
You can put the definition in `Include/cpython/object.h`, rather than use this `#ifdef` here. (See line 825.)

Reply:
See https://devguide.python.org/developer-workflow/c-api/index.html for details on this.